app/scripts/lib/translator.js

```diff
@@ -88,6 +88,19 @@ define(function (require, exports, module) {
       return (templateText) => {
         return this.get(forceText || templateText, context);
       };
+    },
+
+    /**
+     * Resets the translated values to null
+     *
+     * Useful in tests to avoid failures when
+     * running tests with a non-EN-US locale.
+     * @private
+     */
+    _clearTranslationValues () {
+      Object.keys(this.__translations__).forEach((translationKey) => {
```

Comment:
I think you should be able to just do a `this.__translations__ = {}`. `get` will see no translation exists for the key and use the key as the translation.